### PR TITLE
Update error message to indicate if a directory doesn't exist to dump coverage file

### PR DESCRIPTION
**Pull request by saurasm**
Modified function `get_target_output_writable` in `src/output.rs`.
Updated error message indicates if a directory doesn't exist for dumping the coverage file.

### CODE REVIEW

File: src/output.rs [hunk 1]
```diff
@@ -50,14 +50,14 @@ pub fn get_target_output_writable(output_file: Option<&Path>) -> Box<dyn Write>
                 if let Some(parent_path) = parent {
                     if !parent_path.exists() {
                         panic!(
-                            "Cannot create {} to dump coverage data, as {} doesn't exist",
+                            "Cannot create file {} to dump coverage data, as directory {} doesn't exist",
```
Comment (marco-c) [inline src/output.rs:53]:
```suggestion
                            "Cannot create file {} to dump coverage data, as the parent directory {} doesn't exist",
```

Comment (marco-c) [reply src/output.rs:53]: I wonder if we can't just create the directory in this case

File: src/output.rs [hunk 2]
```diff
@@ -50,14 +50,14 @@ pub fn get_target_output_writable(output_file: Option<&Path>) -> Box<dyn Write>
...
                             output.display(),
                             parent_path.display()
                         )
                     }
                 }
                 panic!(
-                    "Cannot create the file {} to dump coverage data.",
+                    "Cannot create file {} to dump coverage data, as the directory doesn't exist",
```
Comment (marco-c) [inline src/output.rs:60]: In this case the error must be due to something else, as we cover the directory not existing case above

File: src/output.rs [hunk 3]
```diff
@@ -50,14 +50,14 @@ pub fn get_target_output_writable(output_file: Option<&Path>) -> Box<dyn Write>
...
                         panic!(
-                            "Cannot create {} to dump coverage data, as {} doesn't exist",
+                            "Cannot create file {} to dump coverage data, as the parent directory {} doesn't exist.",
                             output.display(),
                             parent_path.display()
                         )
                     }
                 }
                 panic!(
-                    "Cannot create the file {} to dump coverage data.",
```
Comment (marco-c) [inline src/output.rs:60]: Could you revert the change on this line?

Comment (saurasm) [reply src/output.rs:60]: I already reverted that change. Line 60 in PR matches with [line 60 in the master branch](https://github.com/mozilla/grcov/blob/725054c044a59138e4670f10cf221d9d310e226d/src/output.rs#L60).

Comment (marco-c) [reply src/output.rs:60]: There is a diff here, it looks like you removed "the "

Comment (saurasm) [reply src/output.rs]: Sorry about that, I've added "the".

**Review by marco-c — APPROVED** [review]
Thanks!
